src/components/ErrorShow/ErrorShow.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import classNames from 'classnames';
+import { ErrorType } from '../../types/ErrorType';
+import './ErrorHide.scss';
+
+interface Props {
+  errorToShow: ErrorType;
+  hideError: () => void;
+}
+
+enum ErrorMessage {
+  add = 'Unable to add a todo',
+  delete = 'Unable to delete a todo',
+  update = 'Unable to update a todo',
+  none = '',
+  emptyTitle = 'Title can\'t be empty',
+}
+
+export const ErrorShow: React.FC<Props> = ({
+  errorToShow,
+  hideError,
+}) => {
+  const handleHideError = () => {
+    hideError();
+  };
+
+  return (
+    <div
+      className={classNames(
+        'notification',
+        'is-danger',
+        'is-light',
+        'has-text-weight-normal',
+        'hidden',
```

Comment:
The 'hidden' class should be conditionally applied based on the errorToShow prop, not hardcoded. You can use a ternary operator to conditionally apply the class.